scripts/apps/packaging/views/sd-package-items-edit.html

```diff
@@ -2,7 +2,7 @@
     <ul class="package-edit" sd-sort-package-items>
         <li ng-repeat="group in list">
             <div class="group-info">
-                <h6>{{ group.id }}</h6>
+                <h6>{{ getPackageLabel(group.id) }}</h6>
```

Comment:
It'd be good to avoid the separation of groups and their translations if we can. Could you instead use
`<h6>{{group.label}}</h6>`

and initialize the translation at the same place where the group is initialized?